documentation/assemblies/assembly-kafka-jmx-remote.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-jmx-options-{context}']
+
+= JMX Remote
+
+{ProductName} supports obtaining JMX metrics from the Kafka brokers. You can obtain various metrics
+about each Kafka broker, for example, usage data such as the `BytesPerSecond` value or the request rate of the network
+of the broker. {ProductName} supports opening a password and username protected JMX port or a non-protected JMX port.
```

Comment:
We try to follow the rule of one sentence per line so that it's simpler to review.